ctutil/sctcheck/sctcheck.go

```diff
@@ -114,9 +114,14 @@ func checkChain(ctx context.Context, lf logInfoFactory, chain []*x509.Certificat
 	}
 
 	var issuer *x509.Certificate
-	for _, c := range chain[1:] {
+	for i := 1; i < len(chain); i++ {
```

Comment:
nit: could do `for i, c := range(chain)` and save a line :)

Reply:
That's why I took this longer approach; it makes sure that `i` is consistent with the index in the outer chain, and it's safer than the slice approach I did first in case the slice I'm slicing into isn't long enough. I'll take the extra line here for that safety :-)